ethportal-api/src/types/consensus/beacon_block.rs

```diff
@@ -174,6 +174,17 @@ impl SignedBeaconBlock {
             SignedBeaconBlock::Deneb(block) => block.message.slot,
         }
     }
+
+    /// Returns execution block number.
+    pub fn execution_block_number(&self) -> u64 {
```

Comment:
This would make us do a match on `ExecutionPayload` every time we want to get the block number, because each fork has its own `ExecutionPayload` type.

match executionpayload {
bellatrix => number
capella => number
deb => number
}

I think the current implementation is the cleanest, but if this is a major concern we can discuss it more